paasta_tools/cli/utils.py

```diff
@@ -796,6 +759,19 @@ def get_instance_configs_for_service(service, soa_dir):
                 soa_dir=soa_dir,
                 load_deployments=False,
             )
+        for _, instance in get_service_instance_list(
```

Comment:
Hopefully this doesn't provide an adhoc job to  something that isn't expecting it. (status? validate? check?)